modules/solid_mechanics/tests/generalized_plane_strain/tests

```diff
@@ -1,38 +0,0 @@
-[Tests]
```

Comment:
As we implement things in tensor_mechanics, we are still keeping all of the tests in solid_mechanics. We generally want to keep all tests around until we get rid of the code. The solid_mechanics code is being widely used, and will be for some time, so we want to make sure it stays working.